lib/importmap/engine.rb

```diff
@@ -7,6 +7,9 @@ module Importmap
   class Engine < ::Rails::Engine
     config.importmap = ActiveSupport::OrderedOptions.new
     config.importmap.sweep_cache = Rails.env.development? || Rails.env.test?
+    config.importmap.rescuable_asset_errors = [
+      defined?(Sprockets::Rails) && Sprockets::Rails::Helper::AssetNotFound,
```

Comment:
This only works if import-maps is loaded before sprockets-rails.

Reply:
Did you mean _after_ sprockets-rails? I think if importmap-rails was loaded before sprockets-rails this _would not_ work, i.e. `defined?(Sprockets::Rails)` would be `false` even though it's going to be defined later once sprockets-rails gets `require`d.

So there does seem to be a load order dependency here. In the past, I've always worked on applications where `sprockets/railtie` was required before `Bundler.require(*Rails.groups)` was invoked, because that's how rails has generated it...but that won't be true in rails 7 after https://github.com/rails/rails/pull/43261. I think we can make it work independent of the load order by inserting into this list within an `initializer` block.

Reply:
Addressed in https://github.com/rails/importmap-rails/compare/34defe58e2cdfa4fe52d03821083da1d0fd57e68..313e2cf3407b83ca6c19f138389f6640efeb6cb3